URBNCreativeTestRig/TextDetailTogglesViewController.swift

```diff
@@ -0,0 +1,359 @@
+//
+//  TextDetailTogglesViewController.swift
+//  URBNCreativeTestRig
+//
+//  Created by Jason Grandelli on 11/9/15.
+//  Copyright © 2015 Jason Grandelli. All rights reserved.
+//
+
+import UIKit
+
+class OptionTextField: UITextField {
+    var setTextBlock: () -> (Void) = {}
```

Comment:
Very good point. Though, I do like the idea of these sort of change callbacks returning what changed.